AVRIoT.X/mcc_generated_files/cli/cli.c

```diff
@@ -211,15 +211,15 @@ static void set_wifi_auth(char *ssid_pwd_auth)
     switch (params)
     {
         case WIFI_PARAMS_OPEN:
-                strncpy(ssid, credentials[0],MAX_WIFI_CREDENTIALS_LENGTH-1);
```

Comment:
the MAX_WIFI_CREDENTIALS_* symbols defined in credentials_storage.h are incorrect and in fact duplicates of the correct symbols defined in the WINC driver (M2M_MAX_*)